app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ProductBrowsingListActivity.java

```diff
@@ -87,6 +129,11 @@ protected void onCreate(Bundle savedInstanceState) {
         searchType = extras.getString(SEARCH_TYPE);
```

Comment:
I think in this case we should always set search type to search as it would not work well for categories, labels, stores etc.

Reply:
I think so, but i let this because there is a switch statement on this variable in this activity, so it will maybe used to do another functionality in the future, but until this time the search type variable always set  to SEARCH.

Reply:
Fine then. :)